addons/website/tests/test_session.py

```diff
@@ -0,0 +1,21 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
+from odoo.tests import HttpCase, tagged
+
+
+@tagged('-at_install', 'post_install')
+class TestSession(HttpCase):
+
+    def test_01_session(self):
+        self.authenticate('admin', 'admin')
+        r = self.url_open('/')
+
+        for line in r.text.splitlines():
+            _, match, session_info_str = line.partition('odoo.__session_info__ = ')
+            if match:
+                session_info = json.loads(session_info_str[:-1])
+                self.assertEqual(session_info['uid'], self.env.ref('base.user_admin').id)
+                return
+
+        raise ValueError('Session info not found in website page')
```

Comment:
I'm not sure this test has a real added value. You basically assert that the uid key is in session_info, but we'll never remove keys from that dict without grepping the whole codebase to see if there're used or not.

Reply:
I will remove it then